src/containerapp/azext_containerapp/commands.py

```diff
@@ -49,6 +50,11 @@ def load_command_table(self, _):
         g.custom_command('create', 'create_containerapp', supports_no_wait=True, exception_handler=ex_handler_factory(), table_transformer=transform_containerapp_output)
         g.custom_command('update', 'update_containerapp', supports_no_wait=True, exception_handler=ex_handler_factory(), table_transformer=transform_containerapp_output)
         g.custom_command('delete', 'delete_containerapp', supports_no_wait=True, confirmation=True, exception_handler=ex_handler_factory())
+        g.custom_command('exec', 'containerapp_ssh', is_preview=True, validator=validate_ssh)
```

Comment:
@StrawnSC have we decided to call this exec? instead of SSH - ssh seems to be self-explanatory. "exec - can be execute or executive? Can we at least use execute (full word) if not ssh

Reply:
During the meeting today, Tomer/Anthony seemed to be under the impression we should call it exec (like similar commands `docker exec` and `kubectl exec` which let you execute commands in docker containers/k8s pods)
 @anthonychu looped in the product owner for this feature on an ADO item to ask about what the naming should be